docs/node_scenarios.md

```diff
@@ -76,6 +76,17 @@ How to set up Alibaba cli to run node scenarios is defined [here](cloud_setup.md
 #### VMware
 How to set up VMware vSphere to run node scenarios is defined [here](cloud_setup.md#vmware)
 
+This cloud type uses a different configuration style, see actions below and [example config file](../scenarios/openshift/vmware_node_scenarios.yml)
+
+*vmware-node-terminate, vmware-node-reboot, vmware-node-stop, vmware-node-start*
+
+#### IBMCloud
+How to set up IBMCloud to run node scenarios is defined [here](cloud_setup.md#ibmcloud)
+
+This cloud type uses a different configuration style, see actions below and [example config file](../scenarios/openshift/ibmcloud_node_scenarios.yml)
+
+*ibmcloud-node-terminate, ibmcloud-node-reboot, ibmcloud-node-stop, ibmcloud-node-start
```

Comment:
Maybe also mention how we can chain to run something like node stop and then start using a sample config here? Thoughts?

Reply:
Thoughts on new example here

Reply:
LGTM!